mlibrary/search
# Remove `@emotion/react`

# Overview
In an effort to reduce dependencies, `@emotion/react` has been replaced with vanilla CSS, and is uninstalled.

## Anything else?
`src/modules/resource-access/components` has been reorganized.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Look through the site and see if anything is broken.
